docs/DafnyRef/Specifications.md

```diff
@@ -844,3 +844,58 @@ add:
    requires Valid()
    reads Repr
 ```
+
+## 7.8. Well-formedness of specifications {#sec-well-formedness}
```

Comment:
Ok, in the next version, I reworded the section about assertion batches so that I clarify that implicit assertions are actually the well-formedness of expressions, and changed the links to them. Indeed, it's better. That way, I can keep this title intact. Thanks for this useful feedback.